### ISSUE TITLE
Error during vite.js build on vaadin 24.3.8

### ISSUE THREAD

**Issue by padmalcom** [FlowingCode/AppLayoutAddon#119]
Hi, first of all thank you for all the addons you provide!

When adding the AppLayoutAddon 'maven install' failes afterwards with the following error. The build error disaapears once I remove the dependency.

```
2024-04-03T13:57:39.889+02:00  ERROR  8204   ---   [onPool-worker-1]    c.v.f.s.frontend.TaskRunDevBundleBuild     :  Command `C:\Users\admin\.vaadin\node\node.exe C:\Users\admin\eclipse-workspace\his-ai\node_modules\vite\bin\vite.js build` failed:
vite v5.1.1 building for production...
transforming...
✓150 modules transformed.
error during build:
RollupError: Expression expected
    at getRollupError (file:///C:/Users/jfrei/eclipse-workspace/his-ai/node_modules/rollup/dist/es/shared/parseAst.js:376:41)
    at ParseError.initialise (file:///C:/Users/jfrei/eclipse-workspace/his-ai/node_modules/rollup/dist/es/shared/node-entry.js:11158:28)
    at convertNode (file:///C:/Users/jfrei/eclipse-workspace/his-ai/node_modules/rollup/dist/es/shared/node-entry.js:12898:10)
    at convertProgram (file:///C:/Users/jfrei/eclipse-workspace/his-ai/node_modules/rollup/dist/es/shared/node-entry.js:12218:12)
    at Module.setSource (file:///C:/Users/jfrei/eclipse-workspace/his-ai/node_modules/rollup/dist/es/shared/node-entry.js:14042:24)
    at async ModuleLoader.addModuleSource (file:///C:/Users/jfrei/eclipse-workspace/his-ai/node_modules/rollup/dist/es/shared/node-entry.js:18681:13)
```

**Comment by mlopezFC** [comment]
Hi @padmalcom 
Thanks for reporting about this issue, can you tell us which exact Vaadin version are you using? .. that will help us to reproduce the issue.
Regards.

**Comment by padmalcom** [comment]
Yes it is 24.3.8

**Comment by mlopezFC** [comment]
I tested it and indeed it seems that it stopped to work since a given version, with Vaadin 24.3.8 it is not working anymore (but it works with Vaadin 24.1.2).
I was able to track the problem to the web component and it seems that it is related to an old lit and typescript version being used. If everything goes well my plan is to release a new version of the web component and then try again with this addon.
I will keep this issue posted on how it goes.
